### ISSUE TITLE
[Bug] Unable to run example code.

### ISSUE THREAD

**Issue by brianleect** [jeasonstudio/solidity-antlr4#1]
```
import { parse } from 'solididty-antlr4';

const code = `// SPDX-License-Identifier: MIT
pragma solidity ^0.8.20;

contract HelloWorld {
  string public greet = "Hello World!";
}
`;

const { parseTree, syntaxTree: ast, errors } = parse(code);
// SyntaxTree: { type: "SourceUnit", nodes: [...], rage: [0, 136], location: {...} }
```

Also, there's a typo i believe it should be `solidity` instead of `solididty`.

After fixing it I'm still unable to run getting the error when running `node main.js` . 

```
import { parse } from 'solidity-antlr4';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1153:20)
    at Module._compile (node:internal/modules/cjs/loader:1205:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47

Node.js v20.9.0
```
After setting package.json type as `module` to resolve that error

```
import { parse } from 'solidity-antlr4';
         ^^^^^
SyntaxError: Named export 'parse' not found. The requested module 'solidity-antlr4' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'solidity-antlr4';
const { parse } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:131:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:213:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.9.0
```

After changing to use 
```
import pkg from 'solidity-antlr4';
const { parse } = pkg;
```

The following error then occurs.

```
(node:11672) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
Error parsing Solidity code: C:\Users\trans\OneDrive - Nanyang Technological University\Desktop\Github repos\web3\gpt-auditor\node_modules\solidity-antlr4\dist\esm\index.js:1
export * from "./grammar";
^^^^^^

SyntaxError: Unexpected token 'export'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1153:20)
    at Module._compile (node:internal/modules/cjs/loader:1205:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at cjsLoader (node:internal/modules/esm/translators:284:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:234:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
```

Changing file to use .mjs did not resolve the error as well. 

Thanks for taking a look at it and please do let me know if I'm trying to use this wrongly lol

**Comment by jeasonstudio** [comment]
Thank you very much for raising this issue. It because `antlr4ng` is a pure ECMAScript module (ESM) package, but `solidity-antlr4` can not be as a pure package (see [this gist](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c)).

Unfortunately, all my projects have webpack or other bundlers that have addressed this issue for me (including the solidity-antlr4 project itself), so I was not aware of it. I will provide a new minor version to support running in a pure Node.js environment, and I will also include some documentation and examples.

**Comment by jeasonstudio** [comment]
@brianleect I have released version [2.2.0](https://www.npmjs.com/package/solidity-antlr4), and now `solidity-antlr4` can run smoothly in Node.js environments with version >= 14. If there are any other issues, please feel free to reach out to me.

Here's an example:
[https://stackblitz.com/edit/stackblitz-starters-vtytk3?file=index.mjs](https://stackblitz.com/edit/stackblitz-starters-vtytk3?file=index.mjs)

**Comment by brianleect** [comment]
Thanks @jeasonstudio , the example works. I tried a bunch of other solidity parsers as well prior for some code I retrieved from etherscan to test and so far it seems that only yours work for me. Great work on this project!
